### ISSUE TITLE
Unable to install Micromamba v2.0.0 and above using Miniconda

### ISSUE THREAD

**Issue by NilaBlueshirt** [mamba-org/mamba#3530]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### Anaconda default channels

- [X] I do NOT use the Anaconda default channels (pkgs/* etc.)

### How did you install Mamba?

Micromamba

### Search tried in issue tracker

libmamba

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

Not applicable

### Describe your issue

Dear Mamba Developers,

Thanks a ton for all of your work! And I apologize if this is something that already been asked/fixed. 

I'm trying to install Mamba v2.0.0 and above using Miniconda, to a linux x86_64 cluster. The method I have been using worked great in the past, including mamba v1.5.10, and here is how:

## 1. Install Miniconda 
by running a freshly downloaded `Miniconda3-latest-Linux-x86_64.sh`

## 2. Install Mamba
`conda install -yc conda-forge mamba=2.0.0`

## 3. Install Micromamba
`packages=(
  numpy
  scipy
  pandas
  natsort
  scikit-learn
  networkx
  beautifulsoup4
  opencv
  geopandas
  sympy
  mpmath
  matplotlib
  plotly
  bokeh
  seaborn
  ipython
  tqdm
  zstandard
  pyarrow
  feather-format
  pyyaml
  yaml
  openpyxl
  xlrd
  dask
  micromamba
  openssh
  rsync
)`
`mamba install -yc conda-forge "${packages[@]}"`
`mamba clean --all -y`

## And here are the error messages after step 1:

Retrieving notices: ...working... done
Channels:
 - conda-forge
 - defaults
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: / warning  libmamba Added empty dependency for problem type SOLVER_RULE_UPDATE
failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - nothing provides libmamba 1.5.6 hfe524e5_0 needed by libmambapy-1.5.6-py312h2dafd23_0

Could not solve for environment specs
The following packages are incompatible
├─ conda-libmamba-solver is installable with the potential options
│  ├─ conda-libmamba-solver [24.1.0|24.7.0|24.9.0] would require
│  │  └─ libmambapy [>=1.5.6,<2.0.0a0 |>=1.5.6,<2.0a0 ] with the potential options
│  │     ├─ libmambapy 1.5.8 would require
│  │     │  └─ libmamba 1.5.8 hfe524e5_2, which can be installed;
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.9] would require
│  │     │  └─ python >=3.10,<3.11.0a0 , which can be installed;
│  │     ├─ libmambapy [1.4.1|1.4.2|...|1.5.9] would require
│  │     │  └─ python >=3.11,<3.12.0a0 , which can be installed;
│  │     ├─ libmambapy 1.5.10 would require
│  │     │  └─ libmamba 1.5.10 h4cc3d14_0, which can be installed;
│  │     ├─ libmambapy 1.5.10 would require
│  │     │  └─ libmamba 1.5.10 hf72d635_1, which can be installed;
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.9] would require
│  │     │  └─ python >=3.9,<3.10.0a0 , which can be installed;
│  │     ├─ libmambapy 1.5.6 would require
│  │     │  └─ libmamba 1.5.6 had39da4_0, which can be installed;
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.8] would require
│  │     │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  │     ├─ libmambapy 1.5.7 would require
│  │     │  └─ libmamba 1.5.7 had39da4_0, which can be installed;
│  │     ├─ libmambapy 1.5.8 would require
│  │     │  └─ libmamba 1.5.8 had39da4_0, which can be installed;
│  │     ├─ libmambapy 1.5.9 would require
│  │     │  └─ libmamba 1.5.9 h4cc3d14_0, which can be installed;
│  │     ├─ libmambapy 1.5.6 would require
│  │     │  └─ libmamba 1.5.6 hfe524e5_0, which does not exist (perhaps a missing channel);
│  │     ├─ libmambapy 1.5.8 would require
│  │     │  └─ libmamba 1.5.8 hfe524e5_0, which can be installed;
│  │     ├─ libmambapy 1.5.8 would require
│  │     │  └─ libmamba 1.5.8 hfe524e5_1, which can be installed;
│  │     ├─ libmambapy 1.5.8 would require
│  │     │  └─ libmamba 1.5.8 hfe524e5_3, which can be installed;
│  │     └─ libmambapy 1.5.8 would require
│  │        └─ python >=3.13,<3.14.0a0 , which can be installed;
│  ├─ conda-libmamba-solver [22.3.0|22.3.1] would require
│  │  └─ libmambapy 0.22.*  with the potential options
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.9], which can be installed (as previously explained);
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.9], which can be installed (as previously explained);
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.8], which can be installed (as previously explained);
│  │     └─ libmambapy [0.22.0|0.22.1] would require
│  │        └─ python >=3.7,<3.8.0a0 , which can be installed;
│  ├─ conda-libmamba-solver [22.12.0|22.6.0|22.8.1|23.1.0] would require
│  │  └─ conda [>=22.11.0,<23.1.0a |>=22.11.0,<23.2.0a |>=4.13,<22.11.0a ] with the potential options
│  │     ├─ conda [22.11.0|22.11.1|...|4.14.0] would require
│  │     │  └─ python >=3.10,<3.11.0a0 , which can be installed;
│  │     ├─ conda [22.11.1|22.9.0|23.1.0] would require
│  │     │  └─ python >=3.11,<3.12.0a0 , which can be installed;
│  │     ├─ conda [22.11.0|22.11.1|...|4.14.0] would require
│  │     │  └─ python >=3.7,<3.8.0a0 , which can be installed;
│  │     ├─ conda [22.11.0|22.11.1|...|4.14.0] would require
│  │     │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  │     └─ conda [22.11.0|22.11.1|...|4.14.0] would require
│  │        └─ python >=3.9,<3.10.0a0 , which can be installed;
│  ├─ conda-libmamba-solver [23.11.0|23.11.1|23.12.0] would require
│  │  └─ libmambapy >=1.5.3,<2.0.0a0  with the potential options
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.9], which can be installed (as previously explained);
│  │     ├─ libmambapy [1.4.1|1.4.2|...|1.5.9], which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.10, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.10, which can be installed (as previously explained);
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.9], which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.6, which can be installed (as previously explained);
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.8], which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.7, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.9, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.6, which cannot be installed (as previously explained);
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.3 would require
│  │     │  └─ libmamba 1.5.3 had39da4_1, which can be installed;
│  │     ├─ libmambapy 1.5.3 would require
│  │     │  └─ libmamba 1.5.3 had39da4_2, which can be installed;
│  │     ├─ libmambapy 1.5.4 would require
│  │     │  └─ libmamba 1.5.4 had39da4_0, which can be installed;
│  │     ├─ libmambapy 1.5.5 would require
│  │     │  └─ libmamba 1.5.5 had39da4_0, which can be installed;
│  │     └─ libmambapy 1.5.3 would require
│  │        └─ libmamba 1.5.3 haf1ee3a_0, which can be installed;
│  ├─ conda-libmamba-solver [23.3.0|23.5.0|...|23.9.3] would require
│  │  └─ libmambapy [>=1.4.1,<2.0.0a0 |>=1.5.1,<2.0.0a0 ] with the potential options
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.9], which can be installed (as previously explained);
│  │     ├─ libmambapy [1.4.1|1.4.2|...|1.5.9], which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.10, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.10, which can be installed (as previously explained);
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.9], which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.6, which can be installed (as previously explained);
│  │     ├─ libmambapy [0.22.0|0.22.1|...|1.5.8], which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.7, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.9, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.6, which cannot be installed (as previously explained);
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.8, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.3, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.3, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.4, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.5, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.3, which can be installed (as previously explained);
│  │     ├─ libmambapy 1.5.2 would require
│  │     │  └─ libmamba 1.5.2 had39da4_1, which can be installed;
│  │     └─ libmambapy 1.5.1 would require
│  │        └─ libmamba 1.5.1 haf1ee3a_0, which can be installed;
│  ├─ conda-libmamba-solver [22.12.0|22.8.1|...|23.9.3] would require
│  │  └─ python >=3.10,<3.11.0a0 , which can be installed;
│  ├─ conda-libmamba-solver [22.12.0|22.8.1|23.1.0] would require
│  │  └─ python >=3.7,<3.8.0a0 , which can be installed;
│  ├─ conda-libmamba-solver [22.12.0|22.8.1|...|23.9.3] would require
│  │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  ├─ conda-libmamba-solver [22.12.0|22.8.1|...|23.9.3] would require
│  │  └─ python >=3.9,<3.10.0a0 , which can be installed;
│  └─ conda-libmamba-solver [23.1.0|23.11.0|...|23.9.3] would require
│     └─ python >=3.11,<3.12.0a0 , which can be installed;
├─ mamba 2.0.2**  is not installable because it requires
│  └─ libmamba 2.0.2 hf72d635_0, which conflicts with any installable versions previously reported;
└─ pin-1 is not installable because it requires
   └─ python 3.12.* , which conflicts with any installable versions previously reported.

build-2024-10-13.sh: line 65: mamba: command not found
build-2024-10-13.sh: line 66: mamba: command not found
build-2024-10-13.sh: line 68: /packages/apps/mamba/2.0.2/bin/activate

### mamba info / micromamba info

```shell
Since the install failed, system won't recognize "mamba".
```

### Logs

```shell
Since the install failed, system won't recognize "mamba".
```

### environment.yml

```yaml
N/A
```

### ~/.condarc

```yaml
channel_priority: flexible
channels:
  - defaults
## DO NOT DO THIS GUYS SPECIFY THE CHANNEL AT RUNTIME ...
# - https://conda.anaconda.org/gurobi
# - http://conda.anaconda.org/gurobi

(I don't actually use this .condarc file though.)
```

**Comment by jjerphan** [comment]
Hi,

This is a duplicate of https://github.com/mamba-org/mamba/issues/3503 see https://github.com/mamba-org/mamba/issues/3503#issuecomment-2397307533 for a temporary solution using micromamba.
